draft/2022-07-27-this-week-in-rust.md

```diff
@@ -138,12 +132,6 @@ Rusty Events between 2022-07-27 - 2022-08-24 🦀
 
 ### Europe
 
-* 2022-07-20 | Warsaw, PL | [Rust Warsaw](https://www.meetup.com/rust-warsaw/)
-    * [**Rust Warsaw #5**](https://www.meetup.com/rust-warsaw/events/287093615/)
-* 2022-07-21 | Oslo, NO | [Rust Oslo](https://www.meetup.com/Rust-Oslo/)
-    * [**Rust & Relax @ Kulturhuset**](https://www.meetup.com/Rust-Oslo/events/287184674/)
-* 2022-07-21 | Wrocław, PL | [Rust Wrocław](https://www.meetup.com/rust-wroclaw/)
-    * [**Rust Wrocław Meetup #27**](https://www.meetup.com/rust-wroclaw/events/287023750/)
 * 2022-07-27 | Amsterdam, NL | [Rust Developers Amsterdam Group](https://www.meetup.com/rust-amsterdam-group/)
     * [**Rust Developers Amsterdam Group**](https://www.meetup.com/rust-amsterdam-group/events/287019877/)
 * 2022-07-28 | Copenhagen, DK | [Copenhagen Rust Group](https://cph.rs/)
```

Comment:
For some reason, Github won't let me select multiple lines to edit for additions that are not changed from the target. This suggestion might need to be made manually. 
```suggestion
* 2022-07-28 | Copenhagen, DK | [Copenhagen Rust Group](https://cph.rs/)
    * [**Hack Night #27**](https://www.meetup.com/copenhagen-rust-meetup-group/events/287057762/)
* 2022-07-28 | Oslo, NO | [Rust Oslo](https://www.meetup.com/Rust-Oslo/)
    * [**Rust & Relax @ Brygg w/Jon Gjengset**](https://www.meetup.com/rust-oslo/events/287422914/)
```

Reply:
Added manually